redbot/cogs/warnings/warnings.py

```diff
@@ -76,6 +83,43 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(_("Custom reasons have been disabled."))
 
+    @warningset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
+        """Toggle warns being sent to the users DM's"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_dm()
+        await self.config.guild(guild).toggle_dm.set(not toggle)
+        toggle = not toggle
+        if toggle == True:
+            await ctx.send("Warns will now be sent to users DM's")
+        else:
+            await ctx.send("Warns will not be sent to users DM's")
+
+    @warningset.command()
+    @commands.guild_only()
+    async def warnchannel(self, ctx: commands.Context, channel: discord.TextChannel):
+        """Set the channel that warns get set to"""
+        guild = ctx.guild
+        try:
+            await self.config.guild(guild).warn_channel.set(channel.id)
+            await ctx.send(f"The warn channel has been set to {channel.mention}")
+        except commands.BadArgument:
+            return await ctx.send("Couldn't find that channel.")
+
+    @warningset.command()
+    @commands.guild_only()
+    async def togglechannel(self, ctx: commands.Context):
+        """Toggle if the warn should be sent to a channel"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_channel()
+        await self.config.guild(guild).toggle_channel.set(not toggle)
+        toggle = not toggle
+        if toggle == True:
```

Comment:
You can change it to just:
```suggestion
        if toggle:
```
Or even not assign line above and just do `if not toggle:`